Tests/SwiftSyntaxBuilderTest/ExtensionDeclTests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class ExtensionDeclTests: XCTestCase {
+  func testExtensionDecl() {
+    let leadingTrivia = Trivia.garbageText("␣")
+
+    let keywords = ["associatedtype", "class"].map { keyword -> VariableDecl in
```

Comment:
I could add a `Token` struct with some of the properties that the [Token.py](https://github.com/apple/swift/blob/main/utils/gyb_syntax_support/Token.py) have. 
And then make an array with a few more tokens? 

Or is this good enough as an example?

Reply:
I’m not sure I fully understand what you mean. For this test case, I think a list of strings is totally sufficient. Just generating more tokens doesn’t really give us any benefit, I’d say. Or did you think of something else?